docs/endpoints/posts.md

```diff
@@ -210,6 +211,8 @@ Deprecated params:
   The Campaign ID of the campaign that the user's post is associated with.
 - **action**: (string) required without action_id.
   Describes the bucket the action is tied to. A campaign could ask for multiple types of actions throughout the life of the campaign.
+  - **type**: (string) required without action_id.
```

Comment:
I think this snuck in here in a rebase! (It's correctly noted as "required" above.)

Reply:
ah yes, thanks for catching!